.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This file should be deleted.

Reply:
ignored it instead in `,gitignore`, my vscode swift extension keeps regenerating the file